helmfile.d/0700.atlantis.yaml

```diff
@@ -0,0 +1,128 @@
+repositories:
+# Cloud Posse incubator repo of helm charts
+- name: "cloudposse-incubator"
+  url: "https://charts.cloudposse.com/incubator/"
+
+
+releases:
+
+#######################################################################################
+## Atlantis                                                                          ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/cloudposse/atlantis/tree/master/helm/atlantis
+#   - https://github.com/cloudposse/charts/blob/master/incubator/monochart
+#
+- name: "atlantis"
+  namespace: "monitoring"
+  labels:
+    chart: "monochart"
+    component: "monitoring"
+    namespace: "monitoring"
+    vendor: "cloudposse"
+    default: "true"
+  chart: "cloudposse-incubator/monochart"
+  version: "0.7.0"
+  wait: true
+  values:
+    - image:
+        repository: cloudposse/root.cloudposse.co
+        tag: "0.2.0"
+        pullPolicy: IfNotPresent  
+
+      statefulset:
+        enabled: true
+        pod:
+          cmd: atlantis
+          args:
+            - server
+
+      configMaps:
+        default:
+          enabled: true
+          env:
+            ATLANTIS_PORT: "4141"
+            ### Required: ATLANTIS_IMAGE;
+            ATLANTIS_IMAGE: '{{ env "ATLANTIS_IMAGE" }}'
+            ### Required: ATLANTIS_ALLOW_REPO_CONFIG;
+            ATLANTIS_ALLOW_REPO_CONFIG: '{{ env "ATLANTIS_ALLOW_REPO_CONFIG" | default "true" }}'
+            ATLANTIS_REPO_CONFIG: "/default/atlantis.yaml"
```

Comment:
Lets parameterize this as well while we're at it.

Reply:
It has been parametrized here
https://github.com/cloudposse/helmfiles/pull/64/files/dc3e9fa37c660e394437c24af7d1988012d42e7e#diff-b7dc0ff1896a93cd4396f2253c517fa6R128
This variable is only for internal purposes.